native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -47,6 +55,12 @@ extension ViewController: UIPickerViewDelegate, UIPickerViewDataSource {
     func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
         return stationData[row]
     }
+    func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+        let departureStation: String = stationData[departurePicker.selectedRow(inComponent: 0)]
+        let arrivalStation: String = stationData[arrivalPicker.selectedRow(inComponent: 0)]
+        presenter.setDepartureStation(departureStation: departureStation)
+        presenter.setArrivalStation(arrivalStation: arrivalStation)
+    }
```

Comment:
I don't know if there's a way to split to two picker views without redoing the whole class but feels strange to be setting both variables when only one picker changes. Probably too much effort though to split them entirely